ui/models/taskcluster.js

```diff
@@ -15,6 +15,12 @@ export default class TaskclusterModel {
     );
   }
 
+  static async getCredentials({ currentRepo, testMode = false }) {
```

Comment:
There is a [helper function](https://github.com/mozilla/treeherder/blob/cfb765d7b222b8e6a09e08187c0e98c4fb933135/ui/helpers/taskcluster.js#L71) to get the credentials - can you use that one?

Reply:
I need user data, not tc creds, so I am not going this route